### DISCUSSION TITLE
[IOS] Messages sent to ios devices are not delivered.

### DISCUSSION THREAD

**Discussion by alexxsanchezm** [invertase/react-native-firebase#6742] [Q&A]
**SUMMARY:** "Two identical projects, identical setup but different bundle id, the old doesn't work the new one works"

Hi, I have a production app which uses @react-native-firebase/messaging package, but after update the APN's certificates the push messages stop coming to my ios app.

I also update my app to use a new distribution certificate and also had to remove the old APN's certificates since they were expired already, which is my fault, of course. Any ways I set up an APNs Key instead of certificates, but still the messages still not delivered.

I've also read Firebase may be blocking push notifications requests to certain apps. In order to do a simple test, I made a clone of my react-native app and changed the bundle identifier + all the other apple setup for the new bundle id. Strange enough this experiment worked, by using different bundle identifier all push messages were delivered.

**QUESTION:** How can I confirm certain bundle id is been banned?

**Comment by mikehardy** [comment]
With apologies - I'm not sure, I would ask this on stackoverflow (where the firebase team actually offers authoritative support) or in the firebase-ios-sdk repo directly (where you can also get authoritative answers)

**Reply by alexxsanchezm** [reply]
I appreciate your feedback.
But also I had reached support from firebase, they pointed out if my service is getting http status 200, then is very unlikely the app has been banned/limited.
Also have successfully tested sending notifications right to APN's to my iPhone, all with the bundle id I though was banned.

Further troubleshooting I did swap again bundle ids, GoogleService.plist and Provisioning profiles. And all messages were received, not even running yarn install or pod install just recompile.

I'll look at stack forums as you mentioned, thanks.
